pxr/usd/usd/testenv/testUsdValidationError.py

```diff
@@ -0,0 +1,210 @@
+#!/pxrpythonsubst
+#
+# Copyright 2024 Pixar
+#
+# Licensed under the terms set forth in the LICENSE.txt file available at
+# https://openusd.org/license.
+
+import unittest
+
+from pxr import Plug, Sdf, Usd
+
+
+class TestUsdValidationError(unittest.TestCase):
+    def test_create_default_error_site(self):
```

Comment:
Since I have started syncing this one internally, I am addressing these but as an fyi the naming convention we seem to use here is `test_CreateDefaultErrorSite`, and `_VerifyErrorSiteWithLayer`, etc. Refer few examples here: https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/usd/usd/testenv/testUsdStage.py.

Reply:
I'll fix them.

Reply:
Thanks @roggiezhang-nv. I have already updated these internally for this PR.

Reply:
I see. Thanks.